### PR TITLE
Pushing the test to show feature branch functionality

### PR DESCRIPTION
**Pull request by favelavela**
Merging changes to demonstrate how to work with PR

### CODE REVIEW

**Comment by AamirMajeed290** [comment]
Aamir reviews

**Review by AamirMajeed290 — COMMENTED** [review]
Aamir reviews

File: app/src/test/java/example/code/tddexample/SimpleSetTest.java [hunk 1]
```diff
@@ -48,29 +48,29 @@ public void simpleSet_ifSimpleSetDoesNotContainElement_returnFalse() {
...
-//    }
+    @Test
+    public void simpleSet_ifElementRemoved_setDoesNotContainElement() {
+        simpleSet.addElement(1);
+
+        assertTrue(simpleSet.contains(1));
+
+        simpleSet.removeElement(1);
+
+        assertFalse(simpleSet.contains(1));
```
Comment (favelavela) [inline app/src/test/java/example/code/tddexample/SimpleSetTest.java:59]: You are the best developer ever!!!

File: app/src/test/java/example/code/tddexample/SimpleSetTest.java [hunk 2]
```diff
@@ -48,29 +48,29 @@ public void simpleSet_ifSimpleSetDoesNotContainElement_returnFalse() {
...
+    }
+
```
Comment (BF2018) [inline app/src/test/java/example/code/tddexample/SimpleSetTest.java:61]: some comment here

File: app/src/test/java/example/code/tddexample/SimpleSetTest.java [hunk 3]
```diff
@@ -48,29 +48,29 @@ public void simpleSet_ifSimpleSetDoesNotContainElement_returnFalse() {
...
-//    @Test
-//    public void simpleSet_ifElementRemoved_returnNewCount() {
-//        simpleSet.addElement(1);
-//
-//        assertTrue(simpleSet.contains(1));
-//        assertEquals(1, simpleSet.elementCount());
-//
-//        simpleSet.removeElement(1);
-//
-//        assertFalse(simpleSet.contains(1));
```
Comment (JamesS2752) [inline app/src/test/java/example/code/tddexample/SimpleSetTest.java:71]: 2/10

**Review by BF2018 — APPROVED** [review]

**Review by Sulekhareddy — APPROVED** [review]

**Review by AamirMajeed290 — APPROVED** [review]

File: app/src/test/java/example/code/tddexample/SimpleSetTest.java (hunk 3 above)
Comment (JamesS2752) [reply app/src/test/java/example/code/tddexample/SimpleSetTest.java:71]: hi

**Pull request merged**